api/test/wfl/tools/parallel_runner.clj

```diff
@@ -0,0 +1,22 @@
+(ns wfl.tools.parallel-runner
+  (:require [clojure.test :as test]))
+
+(defn -main
+  [& args]
+  (letfn [(go! [& tests]
+            (let [output (new java.io.StringWriter)]
+              (binding [test/*test-out* output
+                        test/*report-counters* (ref test/*initial-report-counters*)]
+                (doseq [t tests] (t))
+                [@test/*report-counters* (str output)])))]
+    (let [ns      (symbol (first args))
+          _       (use ns)
```

Comment:
This is hideous but I need something to call `ns-interns` on later. Even `find-ns` doesn't work if it isn't required or something, so this seems like a fix? I don't know. I just want to point at a namespace by its name and say "please let me call `vals` on you"